packages/fxa-settings/src/components/Settings/FlowRecoveryKeyHint/index.tsx

```diff
@@ -78,7 +78,7 @@ export const FlowRecoveryKeyHint = ({
     navigateForward();
     alertBar.success(
       ftlMsgResolver.getMsg(
-        'flow-recovery-key-success-alert-v2',
+        'flow-recovery-key-success-alert',
```

Comment:
We increment the ids because of new message strings.  What happened in this case?  We reverting back to an old string?

Reply:
I'm not sure what happened here, don't fully recall, but there is no v2 in the fluent file and the fallback text here matches the original fluent string.